.github/workflows/ci.yml

```diff
@@ -458,6 +458,9 @@ jobs:
         run: |
```

Comment:
This change is not going to help:(

We blame ryuk because it is downloaded first. I disabled downloading ryuk in the past and it was still failing on other images.

If we could pre-download images before running tests then we would avoid flakiness of docker registries. Cc @ppalucha

Reply:
> We blame ryuk because it is downloaded first.

you're right.

> I disabled downloading ryuk in the past and it was still failing on other images.

Maybe if we retry long enough to download ryuk we cover for the short unavailability of the docker hub.
So it depends on the unavailability pattern (total vs % requests dropped).

Also, wonder whether testcontainers library does / can do retries for all images it downloads.